tools/drsnoop.py

```diff
@@ -48,22 +52,52 @@
 parser.add_argument("-n", "--name",
                     type=ArgString,
                     help="only print process names containing this name")
+parser.add_argument("-v", "--verbose", action="store_true",
+                    help="show system memory state")
 parser.add_argument("--ebpf", action="store_true",
                     help=argparse.SUPPRESS)
 args = parser.parse_args()
 debug = 0
 if args.duration:
     args.duration = timedelta(seconds=int(args.duration))
 
+
+# vm_stat
+
+vm_stat_addr = ''
+with open(kallsyms) as syms:
+    for line in syms:
+        (addr, size, name) = line.rstrip().split(" ", 2)
+        name = name.split("\t")[0]
+        if name == "vm_stat":
```

Comment:
I remember you mentioned `vm_stat` is defined in 3.x kernel? The minimum bpf support is 4.4. Maybe we do not need this?

Reply:
Centos 7.6 with kernel version 3.10.0-957.el7.x86_64 support bpf, which still use `vm_stat`, so I think we need this.